src/main/java/org/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -102,6 +104,7 @@ public void updateItem(LogEventViewModel event, boolean empty) {
 
                 if ((event == null) || empty) {
                     setGraphic(null);
+                    setContextMenu(null);
```

Comment:
Why this call? What is the behavior if this is removed? I would not show any context menu if something is empty.

Then, in src/main/java/org/jabref/gui/errorconsole/ErrorConsoleViewModel.java, you can remove the `null` check.

Reply:
Makes sense. Removed.